native/dlatrd.go

```diff
@@ -0,0 +1,143 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// Dlatrd reduces nb rows and columns of a real n×n symmetric matrix A to symmetric
+// tridiagonal form. It computes the orthonormal similarity transformation
+//  Q^T * A * Q
+// and returns the matrices V and W to apply to the unreduced part of A. If
+// uplo == blas.Upper, the upper triangle is supplied and the last nb rows are
+// reduced. If uplo == blas.Lower, the lower triangle is supplied and the first
+// nb rows are reduced.
+//
+// a contains the symmetric matrix on entry with active triangular half specified
+// by uplo. On exit, the nb columns have been reduced to tridiagonal form. The
+// diagonal contains the diagonal of the reduced matrix, the off-diagonal is
+// set to 1, and the remaining elements contain the data to construct Q.
+//
+// If uplo == blas.Upper, with n = 5 and nb = 2 on exit a is
+//  ⎡a  a  a v4 v5⎤
```

Comment:
I know that I used these in mat64.Format (note that there is still an issue to make that more general and allow ASCII-only formatting), but please don't use them in godoc; they don't render correctly on a non-zero set of machines.